python/tests/unit/arcticdb/version_store/test_write.py

```diff
@@ -132,4 +133,26 @@ def test_write_non_timestamp_index(lmdb_version_store, index_type, sorted, valid
     info = lib.get_info(symbol)
     assert info["sorted"] == "UNKNOWN"
 
+@pytest.mark.parametrize("dtype", [None, object])
```

Comment:
Parameterizing with `dtype=None` means we use the Pandas inferred type right? It might be clearer (to me at least) to state the type explicitly (eg `np.float64` for this test).